commit 6ebcae4340dd19c556c7de890e90b3bb6cebe97d (refs/changes/66/4266/3)
Author: Trent Mick <trentm@gmail.com>
Date:   2018-06-19T14:59:00-07:00 (1 year, 4 months ago)
    
    TRITON-527 cloudapi GetMachine should document 'credentials=true' param
    Reviewed by: Marsell Kukuljevic <marsell@joyent.com>
    Approved by: Marsell Kukuljevic <marsell@joyent.com>

diff --git a/docs/index.md b/docs/index.md
index 83b3756..2066f55 100644
--- a/docs/index.md
+++ b/docs/index.md
@@ -4327,7 +4327,9 @@ been purged from Triton.
 
 ### Inputs
 
-* None
+**Field**   | **Type** | **Description**
+----------- | -------- | ---------------
+credentials | Boolean  | Whether to include the generated credentials for instances, if present. Defaults to false.
 
 ### Returns
 
